beacon-chain/rpc/prysm/v1alpha1/validator/proposer_attestations.go

```diff
@@ -277,7 +279,14 @@ func (a proposerAtts) sortOnChainAggregates() (proposerAtts, error) {
 		return a, nil
 	}
 
-	return a.sortByProfitabilityUsingMaxCover()
+	// Sort by slot first, then by bit count.
+	slices.SortFunc(a, func(a, b ethpb.Att) int {
+		return cmp.Or(
+			-cmp.Compare(a.GetData().Slot, b.GetData().Slot),
```

Comment:
why are you using a negative here ? don't you want these to be in ascending order. ex:`cmp.Compare(a.GetData().Slot, b.GetData().Slot)` and
`cmp.Compare(a.GetAggregationBits().Count(), b.GetAggregationBits().Count()))`